libs/barista-components/core/src/util/number-util.ts

```diff
@@ -18,3 +18,13 @@
 export function clamp(v: number, min: number = 0, max: number = 100): number {
   return Math.max(min, Math.min(max, v));
 }
+
+/**
+ * Rounding to a specific number of decimal spaces.
+ * TODO: Temporary, because a better solution is needed to avoid rounding issues.
```

Comment:
Is this still a TODO?

Reply:
Yeah, I'm not really sure, I don't have a better solution for now, maybe there is none, or we do not need a better one.

Reply:
Your solution is good it prevents any rounding issues so it is working. In my opinion, you can remove the `TODO`.